"Document: drag event":  example text selection for Firefox users (#10739)
mdn/content: files/en-us/web/api/document/drag_event/index.md

### CSS

```css

> body {
>   user-select: none; /*Prevent the text from getting selectable for Firefox users*/

Reviewer: ```suggestion
body {
  /* Prevent the user selecting text in the example */
  user-select: none;
```
